en/engine/renderable/model-component.md

```diff
@@ -0,0 +1,57 @@
+
+### Instancing batching
+
+The batch through **instancing** is suitable for drawing a large number of dynamic models with the same vertex data. When enabled, drawing will be grouped according to the material and vertex data, and the instanced attributes information will be organized in each group, and then complete the drawing at one time.
+
+For the support and related settings of the skin model, refer to the [Bone Animation Component](../animation/skeletal-animation.md#AboutDynamic-Instancing) documentation.
+
+In addition, instancing also supports custom additional instanced attributes, which can pass more differential data between different instances (such as the difference in appearance of a diffuse color between different characters, or the influence of wind in a large grass field). This requires the support of custom effects. For more detailed instructions, please refer to the [Syntax Guide](../../material-system/effect-syntax.md#Custom-Instanced-Properties) documentation.
+
+### Merge VB batching
+
+Dynamic batching is suitable for drawing a large number of non-skinned dynamic models with low number of faces and different vertex data. When enabled, drawing will be grouped according to the material, and then the vertex and world transformation information will be merged in each frame of each group, and then completed in batches. <sup id="a1">[1](#f1)</sup>.
+
+Operations such as merging vertices per frame introduce a portion of CPU overhead, which is particularly expensive in JavaScript. In addition, it is necessary to remind that the number of __draw calls__ is not as low as possible. <sup id="a2">[2](#f2)</sup>, most good performance is often the result of CPU and GPU load balancing, so when using batch functions, be sure to do more tests to identify performance bottlenecks and do targeted optimization.
+
+## Batch best practices
+
+Generally speaking, the priority of the batch system is: static batching> dynamic instancing> dynamic batching. First of all, we must ensure that the material is unified, under this premise:
+If you are certain that certain models will remain completely unchanged during the game cycle, you can use static batching. If there are a large number of the same model repeated drawing, there is only a relatively controllable small difference between each other, you can use dynamic instancing. If there are a large number of models with very low number of faces but different vertex data, you can consider trying dynamic batching.
+
+---
+
+<b id="f1">[1]</b> Note that currently using uniform to upload the batch-transformed world transformation matrix, taking into account the WebGL standard uniform quantity limit, the current batch draws up to 10 models, so for a large number of same For the material model, the number of drawcalls is expected to be reduced by up to 10 times after opening the batch. [↩](#a1)
```

Comment:
> Note that currently using uniform to upload the batch-transformed world transformation matrix

Note that we currently use uniforms to upload the batched world transformation matrix

> after opening the batch

after enabling VB-merging batching

Reply:
modified, changed formatting slightly also.